pyungo/core.py

```diff
@@ -356,7 +361,10 @@ def calculate(self, data):
             jsonschema.validate(instance=data, schema=self._schema)
         t1 = dt.datetime.utcnow()
         LOGGER.info('Starting calculation...')
-        self._data = Data(data)
+        dt1 = dt.datetime.utcnow()
```

Comment:
Measuring the time taken to instantiate the Data object, so we can suggest the user to disable deepcopying if such time is too high